h/security/permissions.py

```diff
@@ -1,4 +1,14 @@
 class Permission:
+    class Annotation:
+        ADMIN = "annotation:admin"  # Is this used anywhere?
+        READ = "annotation:read"
+        WRITE = "annotation:write"  # Is this granted anywhere?
```

Comment:
I think delete this. It doesn't seem to make any sense, we already have `"annotation:create"` and `"annotation:update"` below

Reply:
Similarly this is done in: https://github.com/hypothesis/h/pull/6784